src/daemon/rpmostreed-sysroot.c

```diff
@@ -600,9 +624,10 @@ sysroot_authorize_method (GDBusInterfaceSkeleton *interface,
       /* The daemon is on the session bus, running self tests */
       authorized = TRUE;
     }
-  else if (g_strcmp0 (method_name, "GetOS") == 0)
+  else if (g_strcmp0 (method_name, "GetOS") == 0 ||
+           g_strcmp0 (method_name, "Reload") == 0)
     {
-      /* GetOS() is always allowed */
+      /* GetOS() and Reload() are always allowed */
```

Comment:
Things are weird now because this is identical to `ReloadConfig` right?  The way I was thinking about this is we wouldn't reread the remotes in `/etc`, but we *would* do the "stat /sysroot for mtime" check.

The rationale I have for this is an admin might be in the middle of editing a config file in `/etc`, and having some other unprivileged software come in and read the half-written config file is exactly the problem we're avoiding by having an explicit `reload` command.

So internally we'd split `reload_sysroot()` from `reload_sysroot_and_config()`?

Reply:
`Reload` only does `sysroot_reload` + signal right? `ReloadConfig` does that *and* rereads config files.

Reply:
OK right, I was slightly confused, it *is* doing `ostree_repo_reload_config` but...eh that's all right.

Reply:
No actually that one will reload the remotes too, right?

Reply:
Ahhh, right. It's confusing because we have *three* configs: repo config, remotes, and `rpm-ostreed.conf`. :) I renamed `sysroot_reload` to be more obvious about this and added some comments! :arrow_down: